spec.html

```diff
@@ -25228,29 +25228,28 @@ <h1>Runtime Semantics: Decode ( _string_, _reservedSet_ )</h1>
                 1. If the code units at index (_k_ + 1) and (_k_ + 2) within _string_ do not represent hexadecimal digits, throw a *URIError* exception.
                 1. Let _B_ be the 8-bit value represented by the two hexadecimal digits at index (_k_ + 1) and (_k_ + 2).
                 1. Set _k_ to _k_ + 2.
-                1. If the most significant bit in _B_ is 0, then
+                1. Let _n_ be the number of leading 1 bits in _B_.
+                1. If _n_ = 0, then
                   1. Let _C_ be the code unit whose value is _B_.
                   1. If _C_ is not in _reservedSet_, then
                     1. Let _S_ be the String value containing only the code unit _C_.
                   1. Else,
-                    1. Let _S_ be the substring of _string_ from index _start_ to index _k_ inclusive.
+                    1. Let _S_ be the String value equal to the substring of _string_ consisting of the code units at indices _start_ (inclusive) through _k_ (inclusive).
```

Comment:
Might want to update this to match https://github.com/tc39/ecma262/pull/2027.

Reply:
I'm ready to resolve the conflicts for whichever of those PRs loses the race.

Reply:
Done.